# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/lprng/no-lpd-killing.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./preremove.linux.sh.orig	Sat Nov 10 16:26:28 2001
+++ ./preremove.linux.sh	Sat Nov 10 16:26:58 2001
@@ -2,14 +2,3 @@
 # preremove.linux.sh,v 1.1 2001/08/21 20:33:17 root Exp
 #
 echo RUNNING preremove.linux.sh
-echo "Stopping lpd server"
-killall -INT lpd
-sleep 2
-if [ -f /etc/redhat-release -a -f /sbin/chkconfig ] ; then
-	/sbin/chkconfig lprng off
-	/sbin/chkconfig --del lprng
-else
-    for i in /etc/rc.d/*/*lprng ; do
-        rm $i;
-    done
-fi
--- ./postinstall.linux.sh.orig	2003-11-14 13:36:02.000000000 -0300
+++ ./postinstall.linux.sh	2003-11-29 10:26:57.000000000 -0300
@@ -68,66 +68,3 @@
 	${INSTALL} -m 755 $f `dirname ${DESTDIR}${LPD_CONF_PATH}`/lpd
 fi
 
-if [ "X$MAKEPACKAGE" != "XYES" -a "$INIT" != no ] ; then
-    echo "Running startup scripts"
-    if [ ! -f $init ] ; then
-        echo "Missing $init";
-		exit 1
-    fi
-    if [ -f /sbin/chkconfig ] ; then
-		echo "Stopping CUPS server"
-		service cups stop || /bin/true
-		service cups-lpd stop || /bin/true
-		echo "Stopping LPD server"
-		service lpr stop || service lprng stop || service lpd stop || /bin/true
-		echo "running chkconfig"
-		(
-		/sbin/chkconfig lpr off
-		/sbin/chkconfig lpd off
-		/sbin/chkconfig lprng off
-		/sbin/chkconfig cups off
-		/sbin/chkconfig cups-lpd off
-		/sbin/chkconfig `basename $init` off
-		)
-		echo "Checking Printcap"
-		${SBINDIR}/checkpc -f
-		echo "Running LPRng Startup Scripts"
-		/sbin/chkconfig --add `basename $init`
-		/sbin/chkconfig --list `basename $init`
-		/sbin/chkconfig `basename $init` on
-		echo "Starting Printer"
-		service `basename $init` start
-		echo "Printer Started"
-    else
-		echo "Stopping server"
-		kill -INT `ps ${PSHOWALL} | awk '/lpd/{ print $1;}'` >/dev/null 2>&1
-		sleep 2
-		echo "Checking Printcap"
-		${SBINDIR}/checkpc -f
-		echo "Starting Printer"
-		sh `dirname ${DESTDIR}${LPD_CONF_PATH}`/lpd start
-		echo "Printer Started"
-		m=`dirname ${DESTDIR}${LPD_CONF_PATH}`/lpd
-		cat <<EOF
-# 
-# You will have to install the run time startup files by hand.
-# The $m file contains the standard startup/shutdown code.
-# You should put this file in the file in common set of rc startup
-# scripts and then make symbolic links to it from the run level
-# directories.
-#
-# You can use the following a template for your installation
-# 
-    initdir=/etc/rc.d
-    cp `dirname ${DESTDIR}${LPD_CONF_PATH}`/lpd  \${initdir}/lpd
-    ln -s ../init.d/lpd \${initdir}/rc0.d/K60lprng
-    ln -s ../init.d/lpd \${initdir}/rc1.d/K60lprng
-    ln -s ../init.d/lpd \${initdir}/rc2.d/S60lprng
-    ln -s ../init.d/lpd \${initdir}/rc3.d/S60lprng
-    ln -s ../init.d/lpd \${initdir}/rc4.d/S60lprng
-    ln -s ../init.d/lpd \${initdir}/rc5.d/S60lprng
-    ln -s ../init.d/lpd \${initdir}/rc6.d/K60lprng
-
-EOF
-	fi;
-fi;
